<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>1650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>251</red>
         <green>251</green>
         <blue>217</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Highlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="HighlightedText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Link">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="LinkVisited">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>142</red>
         <green>142</green>
         <blue>212</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Accent">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>251</red>
         <green>251</green>
         <blue>217</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Highlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="HighlightedText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Link">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="LinkVisited">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>142</red>
         <green>142</green>
         <blue>212</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Accent">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>251</red>
         <green>251</green>
         <blue>217</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>251</red>
         <green>251</green>
         <blue>217</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="HighlightedText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Link">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="LinkVisited">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>142</red>
         <green>142</green>
         <blue>212</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Accent">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>971</width>
      <height>1431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>DATA &amp;&amp; INSIGHTS</string>
     </attribute>
     <widget class="QComboBox" name="comboAttrY">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>580</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboAttrX">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>555</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listBlocked">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>650</y>
        <width>461</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUnblock">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>850</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Unblock</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGenerateCustom">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>560</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate Custom Insights</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBlockSelected">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>850</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Block Selected</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableInsights">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>650</y>
        <width>481</width>
        <height>651</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>630</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>BLOCKED KEYS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExportUseful">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>910</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Export Useful Insights</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSaveBlocked">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>880</y>
        <width>351</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Blocked Keys</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>951</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>DATA LOADING</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>951</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="3">
         <widget class="QPushButton" name="btnFetchJson">
          <property name="text">
           <string>Fetch JSON</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>CSV File</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineJsonUrl"/>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="btnLoadCsv">
          <property name="text">
           <string>Load CSV</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="btnBrowseCsv">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineCsvPath"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>JSON URL</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>560</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>630</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>INSIGHTS</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>530</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>CUSTOM PAIR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGenerateDefault">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>520</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string> Generate OS â†” Study Time Insights </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>580</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y:</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tablePeople">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>951</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>DATA PREVIEW</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>HEATMAP</string>
     </attribute>
     <widget class="QPushButton" name="btnUpdateHeatmap">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Heatmap</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableHeatmap">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>951</width>
        <height>781</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>HEATMAP PREVIEW</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>801</width>
      <height>131</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>170</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>142</red>
          <green>142</green>
          <blue>212</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>170</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>142</red>
          <green>142</green>
          <blue>212</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Jersey M54</family>
      <pointsize>85</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>InsightFinder</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Downloads/170035-200.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
